auditlog/models.py

```diff
@@ -418,7 +418,7 @@ def bulk_related_objects(self, objs, using=DEFAULT_DB_ALIAS):
 try:
     from south.modelsinspector import add_introspection_rules
 
-    add_introspection_rules([], ["^auditlog\.models\.AuditlogHistoryField"])
+    add_introspection_rules([], [r"^auditlog\.models\.AuditlogHistoryField"])
```

Comment:
Already done in PR #197.

Reply:
I think we can close that one.

Reply:
Hmm… I think it is better to merge other's PR instead of redoing it. Check #342 for a quick rebase.

Reply:
you exactly did what I did here :)
The change is already part of this PR. why do we need to have another PR for it?

Reply:
To give credit to the one who did the fix. But if you think it is better to include it in your PR, let's go.